packages/fxa-auth-server/package.json

```diff
@@ -25,7 +25,8 @@
     "format": "prettier --write --config ../../_dev/.prettierrc '**'",
     "gen-keys": "ts-node ./scripts/gen_keys.js; ts-node ./scripts/oauth_gen_keys.js; ts-node ./scripts/gen_vapid_keys.js",
     "write-emails": "node -r ts-node/register ./scripts/write-emails-to-disk.js",
-    "clean-up-old-ci-stripe-customers": "ts-node ./scripts/clean-up-old-ci-stripe-customers.js"
+    "clean-up-old-ci-stripe-customers": "ts-node ./scripts/clean-up-old-ci-stripe-customers.js",
+    "paypal-processor": "NODE_ENV=dev CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts"
```

Comment:
Hard coding `NODE_ENV`? 🤨

Reply:
Oh, a convenience for myself, heh. Should remove that for sure.